examples/factors/README.md

```diff
@@ -1,22 +1,203 @@
 # Factors
 
+[install-rust]: https://www.rust-lang.org/tools/install
+[cargo-risczero]: https://crates.io/crates/cargo-risczero
+[examples-dir]: https://github.com/risc0/risc0/tree/main/examples
+[getting-started]: https://www.risczero.com/docs/
+[discord]: https://discord.com/invite/risczero
+
 The _factors_ example is a minimalistic RISC Zero zkVM proof. The prover demonstrates that they know two nontrivial factors (i.e. both greater than 1) of a number, without revealing what those factors are. Thus, the prover demonstrates that a number is composite -- and that they know the factors -- without revealing any further information about the number.
 
-This example was chosen because it is very straightforward. Implementing the verified multiplication and reporting the result in the receipt involves no complexity beyond what is necessary to run the zkVM at all. We therefore hope this example is a good place to look to see all the steps necessary to use the RISC Zero zkVM without any superfluous problem-specific details.
+To see this example in action, [install Rust](install-rust) and use `cargo run` in this directory to run it:
+```
+cargo run
+```
+
+# Tutorial
+## How to Recreate _Factors_
 
-Choosing a simple example necessarily excludes all of the more complex use cases -- so if you are looking for anything beyond the basics, we recommend looking at other examples in this repository!
+This example was chosen because it is straightforward and is a good introduction for beginners. If you're looking to get started creating RISC Zero zkVM projects, you're in the right place! We'll spend the rest of this readme walking you through how to recreate the factors example for yourself, assuming no prior knowledge of RISC Zero.
```

Comment:
"was chosen" for what? 

Perhaps simplify to: "This example is a good introduction for beginners; if you're looking to get started ..."